src/buckets/components/context/csvUploaderProvider.tsx

```diff
@@ -68,12 +68,37 @@ export const CsvUploaderProvider: FC<Props> = React.memo(({children}) => {
   const resetUploadState = (): void =>
     setUploadState(RemoteDataState.NotStarted)
 
+  const handleError = (error: Error): void => {
+    setUploadState(RemoteDataState.Error)
+    reportErrorThroughHoneyBadger(error, {
+      name: 'uploadCsv function',
+    })
+    const message = getErrorMessage(error)
+    dispatch(notify(csvUploaderErrorNotification(message)))
+  }
+
+  const parseTagStringValue = (value: string): string => {
+    // Replaces special characters in accordance to Line Protocol standards
+    // https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#special-characters
+    return value
+      ? value
+          .replace(/,/g, '\\,')
+          .replace(/ /g, '\\ ')
+          .replace(/=/g, '\\=')
+      : '""'
+  }
+
   const uploadCsv = useCallback(
     (csv: string, bucket: string) => {
       setUploadState(RemoteDataState.Loading)
       setTimeout(() => {
         try {
           const {table} = fromFlux(csv)
+          if (!table.length) {
```

Comment:
we may also want to handle whether an error is returned from the parser, something like:

```
if (!table.length || error) {
}
```

where error is defined like so:

```
const {table, error} = fromFlux(csv)